contracts/blender/src/contract.rs

```diff
@@ -46,11 +47,24 @@ mod blender {
         #[ink(topic)]
         recipient: AccountId,
         leaf_idx: u32,
+        new_note: Note,
     }
 
     type Result<T> = core::result::Result<T, BlenderError>;
     type Event = <Blender as ContractEventBase>::Type;
 
+    /// Describes a path from a leaf to the root.
+    ///
+    /// It is represented as a hash sequence from bottom to top:
+    /// - the first will be the 'lefter' of our leaf and its sibling
```

Comment:
Does _lefter_ here is meant to mean "the left element of the two - our leaf and its sibiling"?

Similarly, _righter_ below.

Reply:
our parent has two children - left child and right child; so the 'lefter of `a` and `b`' is the one who is the left child

Reply:
Yes, that's what I meant 👍